floor.go

```diff
@@ -105,19 +108,40 @@ func (f *Floor) watchFloorPrice() {
 				continue
 			}
 
+			// calculate if price has moved up or down
+			var priceRaw float64
+			if priceRaw, err = strconv.ParseFloat(strings.ReplaceAll(price, " SOL", ""), 64); err != nil {
+				logger.Errorf("Error with price format for %s", f.Name)
+				continue
+			}
+			var increase bool
+			if priceRaw >= oldPrice {
```

Comment:
@rssnyder I based this PR on your code in the `token` ticker. I, however, think the logic can be improved a bit. Now the bot only is red for one step, after which it becomes green again because the values are now equal. This results in a mainly green bot. Maybe we should change this to a `>` condition and add a `=` on which it becomes white again. Another option would be to add a DB which stores the last values (e.g. 24 hours) and do the condition based on that while showing a percentage. 🤔 Let me know what you think.

https://github.com/rssnyder/discord-stock-ticker/blob/be61759521235fae8e344b4e03fa39934ee1e5a8/token.go#L181-L185

Reply:
I am not super into the idea of storing cost data in the database. Can we just add a condition to do nothing if the value is equal so it stays with the previous color?